docs/docs/working-with-images.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Working With Images In Gatsby
+---
+
+# Working With Images In Gatsby
+
+Gatsby has several useful [plugins](/docs/plugins/) that work together for adding and optimizing images for [page components](/docs/building-with-components/#page-components)
+
+
+The recommend approach is to 
+* GraphQL to query image files
+* Sharp, an image processing library, to transform images
+* The Gat
```

Comment:
Looks like the third item in this list is missing something?

Reply:
I'm assuming this is a draft so hopefully this isn't an exercise in futility for me to point out things you're already aware of...:)